frameworks/cassandra/tests/test_jmx.py

```diff
@@ -0,0 +1,304 @@
+                        "trust_store_password_file": TRUST_STORE_PASS,
+                    }
+                }
+            },
+            service_options,
+        )
+
+    if authentication:
+        secret_path = foldered_name + "/" + config.SECRET_VALUE
+        create_secret(secret_value=config.SECRET_VALUE, secret_path=secret_path)
+        service_options = sdk_utils.merge_dictionaries(
+            {
+                "service": {
+                    "security": {
+                        "authentication": {
+                            "enabled": True,
+                            "superuser": {"password_secret_path": secret_path},
+                        },
+                        "authorization": {"enabled": True},
+                    }
+                }
+            },
+            service_options,
+        )
+
+    sdk_install.install(
+        config.PACKAGE_NAME,
+        foldered_name,
+        config.DEFAULT_TASK_COUNT,
+        additional_options=service_options,
+    )
+
+
+def install_jmx_secrets():
+    test_run_id = random_string()
+    create_keystore_cmd = [
+        "keytool",
+        "-genkey",
+        "-alias",
+        "self-signed-cert",
+        "-keyalg",
+        "rsa",
+        "-dname",
+        "CN=myhost.example.com,O=Example Company,C=US",
+        "-keystore",
+        "/tmp/{}-self-signed-keystore.ks".format(test_run_id),
+        "-storepass",
+        "deleteme",
+        "-keypass",
+        "deleteme",
+        "-storetype",
+        "jks",
+    ]
+
+    subprocess.check_output(create_keystore_cmd)
+
+    keystore_list_cmd = [
+        "keytool",
+        "-list",
+        "-v",
+        "-keystore",
+        "/tmp/{}-self-signed-keystore.ks".format(test_run_id),
+        "-storepass",
+        "deleteme",
+    ]
+
+    subprocess.check_output(keystore_list_cmd)
+
+    write_to_file("deleteme", "/tmp/{}-keystorepass".format(test_run_id))
+    write_to_file("admin adminpassword", "/tmp/{}-passwordfile".format(test_run_id))
+    write_to_file("admin readwrite", "/tmp/{}-access".format(test_run_id))
+
+    sdk_security.install_enterprise_cli(False)
+
+    sdk_cmd.run_cli(
+        "security secrets create -f /tmp/{}-self-signed-keystore.ks {}".format(
+            test_run_id, KEY_STORE
+        )
+    )
+    sdk_cmd.run_cli(
+        "security secrets create -f /tmp/{}-passwordfile {}".format(test_run_id, PASSWORD_FILE)
+    )
+    sdk_cmd.run_cli(
+        "security secrets create -f /tmp/{}-keystorepass {}".format(test_run_id, KEY_STORE_PASS)
+    )
+    sdk_cmd.run_cli("security secrets create -f /tmp/{}-access {}".format(test_run_id, ACCESS_FILE))
+
+
+def uninstall_jmx_secrets():
+    sdk_security.delete_secret(KEY_STORE)
+    sdk_security.delete_secret(KEY_STORE_PASS)
+    sdk_security.delete_secret(ACCESS_FILE)
+    sdk_security.delete_secret(PASSWORD_FILE)
+
+
+@pytest.mark.sanity
+@sdk_utils.dcos_ee_only
+def secure_jmx_configuration(self_signed_trust_store, authentication):
```

Comment:
Can you rename this to something that's closer to its intent? Like
```
check_secure_jmx_output()
```
or 
```
verify_jmx_cmd_succeeded()
```

Reply:
changed to ```check_secure_jmx_output```